modules/create_timepoint/test/TestPlan.md

```diff
@@ -0,0 +1,44 @@
+# Create Timepoint Test Plan:
+
+1. Access a candidate that does not have all timepoints from project created
+from Access Profile and click "Create time point"
+  [Automation Testing]
+2. Ensure that clicking on all elements of the breadcrums takes you to the 
+appropriate page.
+  [Manual Testing]
+3. Ensure that the DCCID field displays the correct candidate's identifier.
+  [Manual Testing]
+4. Choose a subproject from dropdown and ensure that the page reloads with 
+subproject selected and also a list of visitlabels if <labelSet> is set in the config.xml.
+  [Manual Testing]
+   - Ensure that if the `config.xml` file contains a `<visitLabel subprojectID="X">` 
+   matching the subproject ID selected, a list of visits ()defined in the same file 
```

Comment:
```suggestion
   matching the subproject ID selected, a list of visits defined in the same file 
```

Not sure why the brackets are there? Let me know if they were intentional for whatever reason

Reply:
probably when i was banging my head on the keyboard from that other PR